Formula/h/halide.rb

```diff
@@ -30,39 +29,24 @@ class Halide < Formula
   depends_on "lld"
   depends_on "llvm"
   depends_on "python@3.13"
-
-  # Check wabt version in `dependencies/wasm/CMakeLists.txt`.
-  # TODO: Ask upstream to support usage of a system-provided wabt.
-  # TODO: Do we really need a git checkout here?
-  resource "wabt" do
-    url "https://github.com/WebAssembly/wabt.git",
-        tag:      "1.0.33",
-        revision: "963f973469b45969ce198e0c86d3af316790a780"
-  end
+  depends_on "wabt"
 
   def python3
     "python3.13"
   end
 
   def install
-    builddir = buildpath/"build"
-    (builddir/"_deps/wabt-src").install resource("wabt")
-
     site_packages = prefix/Language::Python.site_packages(python3)
     rpaths = [rpath, rpath(source: site_packages/"halide")]
     args = [
       "-DCMAKE_INSTALL_RPATH=#{rpaths.join(";")}",
       "-DHalide_INSTALL_PYTHONDIR=#{site_packages}",
-      "-DHalide_SHARED_LLVM=ON",
-      "-DPYBIND11_USE_FETCHCONTENT=OFF",
-      "-DFLATBUFFERS_USE_FETCHCONTENT=OFF",
-      "-DFETCHCONTENT_SOURCE_DIR_WABT=#{builddir}/_deps/wabt-src",
-      "-DCMAKE_SHARED_LINKER_FLAGS=-llldCommon",
+      "-DHalide_LLVM_SHARED_LIBS=ON",
+      "-DHalide_USE_FETCHCONTENT=OFF",
```

Comment:
```suggestion
      "-DHalide_USE_FETCHCONTENT=OFF",
      "-DWITH_TESTS=NO",
      "-DWITH_TUTORIALS=NO",
```

Consider additionally disabling tests and tutorials here if you're not going to run them.